categories/websites/rules-to-better-vue.md

```diff
@@ -5,6 +5,7 @@ guid: 30b621b8-7581-444e-81b9-e5a9253d887d
 uri: rules-to-better-vue
 index:
 - why-vue-is-great
+- set-up-vue-project
```

Comment:
Note: Because of https://github.com/SSWConsulting/SSW.Rules.Content/pull/6009 you should also update this category file to avoid conflicts